package.json

```
@@ -4,7 +4,7 @@
     "tailwindcss": "^3.2.6"
   },
   "scripts": {
-    "tailwind-build": "pnpx tailwindcss build -i ./styles.css -o ./dist/output.css",
-    "tailwind-watch": "pnpx tailwindcss -i ./styles.css -o ./dist/output.css --watch"
+    "tailwind-build": "npx tailwindcss build -i ./styles.css -o ./dist/output.css",
+    "tailwind-watch": "npx tailwindcss -i ./styles.css -o ./dist/output.css --watch"
```

```suggestion
    "tailwind-build": "pnpm exec tailwindcss build -i ./styles.css -o ./dist/output.css",
    "tailwind-watch": "pnpm exec tailwindcss -i ./styles.css -o ./dist/output.css --watch"
```

we're using pnpm as our package manager (bec its the fastest) so i spose we should stick with their stuff. `pnpx` is being deprecated in favor of `pnpm exec` (which is built-in to pnpm) tho: https://github.com/woocommerce/woocommerce/issues/32508